pyanaconda/payload/dnfpayload.py

```diff
@@ -907,11 +907,11 @@ def set_updates_enabled(self, state):
 
         # Enable or disable updates.
         if self._updates_enabled:
-            self.enable_repo("updates")
-            self.enable_repo("updates-modular")
+            for repo in constants.DEFAULT_REPOS:
```

Comment:
It should be `DEFAULT_UPDATE_REPOS`, same below.

Reply:
I'll fix this shortly.